.gitignore

```diff
@@ -3,8 +3,7 @@
 .idea/
 out/
 target/
-java/
```

Comment:
@jingming pointed out that it also ignores source files in `src/main/java/`, which isn't ideal. Where do the docs show up in `java` folder?